src/services/models/telemetryModels.js

```diff
@@ -23,10 +23,23 @@ export const toRuleModel = (response = {}) => {
     'action.type': 'type'
   });
   return update(model, {
-    severity: { $set: (model.severity || '').toLowerCase() }
+    severity: { $set: (model.severity || '').toLowerCase() },
+    actions: { $set: (response.Actions || []).map(toActionModel) }
   });
 };
 
+export const toActionModel = (response = {}) => {
```

Comment:
The issue that caused this to be needed should be fixed now.

Reply:
Removed